test/integrations/destinations/ga4/network.ts

```diff
@@ -1,119 +1,139 @@
-export const networkCallsData = [
+const headers = {
+  HOST: 'www.google-analytics.com',
+  'Content-Type': 'application/json',
+};
+
+const params = {
+  api_secret: 'dymmyApiSecret',
+};
+
+const businessMockData = [
```

Comment:
Keep existing mocks and label them join the new mocks with them

Reply:
addressed